meta-refkit/conf/distro/include/refkit-development.inc

```diff
@@ -33,3 +33,4 @@ REFKIT_IMAGE_BUILD_MODE_SELECTED = "1"
 
 REFKIT_IMAGE_COMPUTERVISION_EXTRA_FEATURES_append = " computervision-test"
 REFKIT_IMAGE_FEATURES_COMMON_append = " common-test"
+BAD_RECOMMENDATIONS_remove = "bluez5-client"
```

Comment:
Something is not right because according to ```BAD_RECOMMENDATIONS``` is must work:

```
It is important to realize that if you choose to not install packages using this variable and some other packages are dependent on them (i.e. listed in a recipe's RDEPENDS variable), the OpenEmbedded build system ignores your request and will install the packages to avoid dependency errors.
```